src/rustzcash.rs

```diff
@@ -6,6 +6,12 @@ extern crate pairing;
 extern crate rand;
 extern crate sapling_crypto;
 
+const SPROUT_GROTH16_PARAMS_HASH: &'static str = "7a6723311162cb0c664c742d2fa42278195ade98ba3f21ef4fa02b82c83aed696e107e389ac7b3b0f33f417aeefe5be775d117910a473a422b4a1b97489fbdd6";
+const SAPLING_SPEND_PARAMS_HASH: &'static str = "35f6afd7d7514531aaa9fa529bdcddf116865f02abdd42164322bb1949227d82bdae295cad9c7b98d4bbbb00e045fa17aca79c90f53433a66bce4e82b6a1936d";
+const SAPLING_OUTPUT_PARAMS_HASH: &'static str = "f9d0b98ea51830c4974878f1b32bb68b2bf530e2e0ae09cd2a9b609d6fda37f1a1928e2d1ca91c31835c75dcc16057db53a807cc5cb37ebcfb753aa843a8ac21";
```

Comment:
These are not the hashes I have, after just doing ``zcutil/fetch-params.sh --testnet``.

Reply:
I have the ones that @ebfull posted here: https://chat.zcashcommunity.com/channel/zcashd-team?msg=gKNBrseZN5wR9uFFk

Reply:
These are BLAKE2b hashes (as opposed to the SHA256 hashes used in `fetch-params`.) I didn't want to bring in another Rust dependency this late in the game.